src/modules/ROOT/pages/changelog/202303/05.json

```
@@ -0,0 +1,161 @@
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/1339"
+      ]
+    },
+    {
+      "what": [
+        "case sensitive search in a record list if MySQL database is used."
+      ],
+      "why": [
+        "to make searching of records easier by returning similar results to the searching criteria."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/1355"
+      ]
+    }
+  ],
+  
+  "changed": [
+    {
+      "what": [
+        "the visibility of the inline record editor option in a record list to hidden if a module isn't provided."
+      ],
+      "why": [
+        "to not confuse the user that the option is available and can be used."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/commit/b96ea0ea236eec6903c881529bbcbc9e393220ab"
+      ]
+    },
+    {
+      "what": [
+        "loading of page resources when switching between pages."
+      ],
+      "why": [
+        "to prevent loading page resources for a page that isn't currently selected."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/commit/638ba17c89a03b99588241e3969edd04d3ef1a75"
+      ]
+    },
+    {
+      "what": [
+        "general ui/ux improvements:",
+        "** Adjust the element styles in the topbar of all web applications to improve the UI for current and older browser versions."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/commit/db740f7f28c78cb9b38e6c5918bca6807363b53a"
+      ]
+    }
+  ],
+  "fixed": [
+    {
+      "what": [
+        "overflowing of long texts in {APP_NAME_COMPOSE}."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/commit/895796a4758479ada793228667ea9b1b4618b749"
+      ]
+    },
+    {
+      "what": [
+        "compose sub-pages not being shown when configuring a navigation page block."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/1359"
+      ]
+    },
+    {
+      "what": [
+        "module field labels shown underneath delete button when bulk editing a value."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/commit/6f6df66d6f8dd28b29d581cbbf5443c4864294d4"
+      ]
+    },
+    {
+      "what": [
+        "drill-down option in metric page block not working if the metric label wasn't provided."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/commit/95ae09e80c35ec24f21379a246f9c3c8e2654a5e"
+      ]
+    },
+    {
+      "what": [
+        "removing of unnecessary resources in device memory after switching from one page to another in {APP_NAME_COMPOSE}."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/commit/95cf245daafc521fc25e0f3bb20b5e2e9ed1b94a"
+      ]
+    },
+    {
+      "what": [
+        "not being able to save a page after any page block is removed from a tabs page block."
```

```suggestion
        "not being able to save a page after removing a page block from a tabs page block."
```

I think this sounds a bit cleanner